### PR TITLE
refactor(tables): Put Tables into Atoms

### PR DESCRIPTION
**Pull request by parlarjb**
https://jira.rax.io/browse/FRMW-273

 - [x] Dev LGTM
 - [x] Dev LGTM
 - [x] Design LGTM

screen shot 2016-02-23 at 10 28 31 am

### CODE REVIEW

**Comment by 100stacks** [comment]
LGTM

File: demo/templates/atoms/tables.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
+<rx-page title="'Tables'">
```
Comment (ssalinas24) [inline demo/templates/atoms/tables.html]: Insert `<hr/>` tag after line number 2 to get the horizontal rule line below title.

File: demo/templates/atoms/tables.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  <ul class="list">
```
Comment (ssalinas24) [inline demo/templates/atoms/tables.html]: @glynnis  and @CITguy The mockups don't show content formatted with lists, but previously we have had items in lists. Are we going to keep the content formatted in lists, and ignore mockups?

File: demo/templates/atoms/tables.html (hunk 1 above)
Comment (CITguy) [inline demo/templates/atoms/tables.html]: This would be part of FRMW-711

File: demo/templates/atoms/tables.html (hunk 2 above)
Comment (CITguy) [inline demo/templates/atoms/tables.html]: The mockup seems to have removed the list that currently exists in documentation. I'd rather keep the list.

Comment (glynnis) [inline demo/templates/atoms/tables.html]: @ssalinas24 I think given the rest of the cosmetic/visual changes that will need to be made down the line, it's okay to leave this in a list for now.

**Comment by glynnis** [comment]
Design LGTM

**Comment by CITguy** [comment]
LGTM

**Pull request merged**